app/uk/gov/hmrc/minorentityidentification/config/AppConfig.scala

```diff
@@ -30,4 +30,13 @@ class AppConfig @Inject()(servicesConfig: ServicesConfig) extends FeatureSwitchi
   val graphiteHost: String = servicesConfig.getString("microservice.metrics.graphite.host")
 
   val timeToLiveSeconds: Long = servicesConfig.getInt("mongodb.timeToLiveSeconds").toLong
+
+  lazy val desStubBaseUrl: String = servicesConfig.getString("microservice.services.des.stub-url")
```

Comment:
These two base urls will be read from the application.conf file. However, the DES service is not defined there. Checkout the sole trader or incorporated entity backends.

Reply:
good spot! forgot add before but now added on application.conf